packages/cli/src/cmds/beacon/options.ts

```diff
@@ -97,6 +98,11 @@ export const beaconExtraOptions: CliCommandOptions<BeaconExtraArgs> = {
     description: "Whether to reuse the same peer-id across restarts",
     type: "boolean",
   },
+
+  private: {
+    description: "Do not send implementation and version name over p2p identify protocol and builder requests",
```

Comment:
Potential simplification

```suggestion
    description: "Do not send implementation details over p2p identify protocol and in builder requests",
```